project-fortis-interfaces/src/actions/Admin/index.js

```diff
@@ -7,6 +7,11 @@ function getListAfterRemove(listBeforeRemove, itemsRemoved, keyBy) {
   return differenceBy(listBeforeRemove, itemsRemoved, keyBy);
 }
 
+function addIdsToUsersForGrid(users) {
+  users.map(user => user.id = `${user.identifier}-${user.role}`);
```

Comment:
Using `.map` usually indicates that you're getting a transformed array from the input array. Here you're actually mutating the values in the original array so it might be better to make this explicit and use `.forEach` and not return a new array here.

Alternatively, you can make the loop side-effect-free (and continue using map) if you use `Object.assign` to first copy the original user, but here the `.forEach` with mutation should be fine/simpler.

Basically:

Either use this code that preserves the implicit `.map` contract that you're getting a new array:

```js
function addIdsToUsersForGrid(users) {
  return users.map(user => {
    const newUser = Object.assign({}, user);
    newUser.id = `${user.identifier}-${user.role}`;
    return newUser;
  });
}
```

Or use this code that makes the mutation of the input explicit by not returning a value and using `.forEach`:

```js
function addIdsToUsersForGrid(users) {
  users.forEach(user => user.id = `${user.identifier}-${user.role}`);
}
```